src/components/logging.ts

```diff
@@ -121,6 +120,10 @@ class Logging {
         maxFiles: 5
     */
     configure(config: LoggerConfig = {}) {
+        // `winston-daily-rotate-file` has side-effects so we don't want to mess anyone up
+        // unless they want to use logging
+        require("winston-daily-rotate-file");
```

Comment:
This did fix my problem but https://github.com/matrix-org/matrix-appservice-bridge/issues/243 is still a problem. 

Interestingly, I can still run `Logging.configure({ level: 'silent' });` which imports the problem library anyway.

I haven't tested whether `winston-daily-rotate-file` still does it's job though.

Reply:
I'd suggest importing it at 157, because that's when we invoke DailyRotateFile? Otherwise this is fine.

Reply:
:+1: Fixed up the build as well